tests/Yaapii.Atoms.Tests/Func/BiFuncWithFallbackTest.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using Yaapii.Atoms.Func;
+
+namespace Yaapii.Atoms.Tests.Func
+{
+    public sealed class BiFuncWithFallbackTest
+    {
+        [Fact]
+        public void UsesMainFunc()
```

Comment:
call it "DeliversValue" (we have no "main" func, but we know what the object should do in this test)